content/enterprise/admin/infrastructure/admin.mdx

```diff
@@ -1,16 +1,12 @@
 ---
-page_title: Active/Active - Infrastructure Administration - Terraform Enterprise
+page_title: Infrastructure Administration - Terraform Enterprise
 ---
 
-# Terraform Enterprise Active/Active
+# Terraform Enterprise Admin
 
-When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](/enterprise/install/automated/active-active) for more information on moving to Active/Active.
+The [Active/Active](/enterprise/install/automated/active-active) operational mode disables use of the Replicated Admin Console.  To accommodate this, there are now admin commands to facilitate configuration changes, safe application stops, and producing support bundles. There is a new container in the TFE architecture to support this called "tfe-admin". You must login to a node in the Active/Active cluster using SSH to run these commands from the command line. 
```

Comment:
Do we need to tell folks here about the new container, or can we just tell them how to log in and run the commands? This shows my lack of knowledge of TFE - please let me know if that should be added back in. BUT I made a few edits to the rest to make it flow better!

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line. 
```